test/data/css/tests.js

```diff
@@ -209,6 +210,25 @@ exports.test_data = {
             }
         ]
     }, {
+        name: "Preserve Newlines",
+        description: "",
+        matrix: [{
+            options: [
+                { name: "preserve_newlines", value: "true" }
+            ],
+            separator_input: '\\n\\n',
+            separator_output: '\\n\\n',
+        }, {
+            options: [
+                { name: "preserve_newlines", value: "false" }
+            ],
+            separator_input: '\\n\\n',
+            separator_output: '\\n',
+        }],
+        tests: [
+            { input: '.div {}{{separator_input}}.span {}', output: '.div {}{{separator_output}}.span {}' },
```

Comment:
@TimvdEijnden  - 
Okay, I added a whitespace marked up output for the tests which shows the issue.  
You should be able to see the problem now.